code/game/objects/structures/electricchair.dm

```diff
@@ -91,5 +91,5 @@
 			to_chat(buckled_mob, "<span class='danger'>You feel a deep shock course through your body!</span>")
 			spawn(1)
 				buckled_mob.electrocute_act(110, src, 1)
-	A.power_light = light
+	A.powernet.lighting_powered = light
```

Comment:
Shouldn't need to declare `var/light` if you do this:
```suggestion
	A.powernet.lighting_powered = A.powernet.has_power(PW_CHANNEL_LIGHTING)
```

Reply:
this entire section has been removed, I'm not sure why the electric chair was setting power channels, that's whack